modules/arrow/src/geoarrow/convert-geoarrow-to-binary-geometry.ts

```diff
@@ -13,6 +14,7 @@ export type BinaryDataFromGeoArrow = {
   binaryGeometries: BinaryFeatures[];
   bounds: [number, number, number, number];
   featureTypes: {polygon: boolean; point: boolean; line: boolean};
+  meanCenters?: number[][];
 };
 
 type BinaryGeometryContent = {
```

Comment:
Nit: Add a comment before each type, we have a lot of these, maybe hint at when this type is used?